CHANGELOG.md

```diff
@@ -1,19 +1,36 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.4.0]
+
+### Added
+
+- Add support for Terraform `v0.15`
+
+## 0.3.0 (unreleased)
```

Comment:
I've issued all releases. You could just checkout the commit locally and push a tag which you release later on.